web/models/tp_rating_profile.ex

```diff
@@ -26,17 +24,15 @@ defmodule CgratesWebJsonapi.TpRatingProfile do
   def changeset(struct, params \\ %{}) do
     struct
     |> cast(params, @attributes )
-    |> validate_required([:tpid, :loadid, :direction, :tenant, :category, :subject, :activation_time, :rating_plan_tag])
+    |> validate_required([:tpid, :loadid, :tenant, :category, :subject, :activation_time, :rating_plan_tag])
     |> validate_length(:tpid, max: 64)
     |> validate_length(:loadid, max: 64)
-    |> validate_length(:direction, max: 8)
     |> validate_length(:tenant, max: 64)
     |> validate_length(:category, max: 32)
     |> validate_length(:subject, max: 64)
     |> validate_length(:activation_time, max: 24)
     |> validate_length(:rating_plan_tag, max: 64)
     |> validate_length(:fallback_subjects, max: 64)
-    |> validate_length(:cdr_stat_queue_ids, max: 64)
     |> unique_constraint(:subject, name: :tp_rating_profiles_tpid_loadid_tenant_category_direction_su_key)
```

Comment:
I don't think it is an actual index. I've found only `tp_rating_profiles_tpid_loadid_tenant_category_subject_acti_key` uniq index